core/runtimes/FiberRuntime/index.js

```diff
@@ -85,8 +87,8 @@ FiberRuntime.prototype.runAsyncCallback = function (cb) {
 };
 
 FiberRuntime.prototype.run = function (processes, options, callback) {
-  this._list = processes;
-  this._count = this._list.length;
+  this._processList = _.keyBy(processes, 'name');
```

Comment:
Converts the list of processes into an associative array with the name of the processes as the keys